### ISSUE TITLE
Does gapit have a python version

### ISSUE THREAD

**Issue by yitengfei120011** [jiabowang/GAPIT#142]
Hi, gapit is a good job,does it have a python version ?

**Comment by prabirbioinfo** [comment]
Nope. but there are other tools and libraries in Python that you can use for GWAS  PyGWAS,  scikit-allel, Hail.
